expo-zustand-styled-components/src/components/BranchNavigation/BranchNavigation.tsx

```diff
@@ -14,38 +14,38 @@ import {
   ButtonTextStyled,
 } from './BranchNavigation.styles';
 import { useRepoInfoStore } from '../../hooks/stores';
+import LinkButton from '../LinkButton/LinkButton';
 
-const BranchNavigation = () => {
+const BranchNavigation = (props: { path?: string; branch?: string }) => {
   const { width } = useWindowDimensions();
-  const { path, name, branch } = useRepoInfoStore();
+  const { name, owner, branch } = useRepoInfoStore();
```

Comment:
i belive the owner and name should be gotten from the URL params if we are really moving the stored values to the url